### PR TITLE
Add Home Page back + font issue

### PR DESCRIPTION
**Pull request by martinyis**
This PR fixes issue #77 

I have made some changes to the codebase and would like to submit a pull request for your review.

I have added the back home page feature, which is now visible on the page. Additionally, I have updated the font family in theme.js from "Roboto" to Google Sans. I have also resolved the font issue on the home page.

However, I have noticed that there are some problems with margins due to the typography taking styles from the theme based on their variants h2, h4, and h5. As a result, the new sizes in theme.js, based on Figma files, conflict with the actual margins and paddings in the code. I can address this issue as well if needed.

### CODE REVIEW

**Comment by ArunPatel02** [comment]
@Soumya-Kushwaha can you review these changes.

File: src/theme.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        600: "#666666",
+        700: "#525252",
+        800: "#333333",
+      },
+    },
+    eventBackground: "#F6F6F6",
+    teamsBackground: "#EAEAEA",
+  },
+  typography: {
+    fontFamily: "Roboto",
```
Comment (Yashsharma1911) [inline src/theme.js:36]: Hmm 🤔 we still have Roboto 

```suggestion
    fontFamily: "Open Sans",
```

**Review by Yashsharma1911 — CHANGES_REQUESTED** [review]
Thanks, nice work @martinyis LGTM except for changes that I have suggested

**Review by Yashsharma1911 — APPROVED** [review]
Never mind let's go and merge this PR 😉

**Comment by Yashsharma1911** [comment]
ohh merge conflict @martinyis resolve this before merging

**Pull request closed without merging**